firebase-firestore/src/main/java/com/google/firebase/firestore/util/CustomClassMapper.java

```diff
@@ -1011,4 +1015,18 @@ public String toString() {
       }
     }
   }
+
+  static class DeserializeContext {
+    final ErrorPath errorPath;
```

Comment:
I have a guess what this is for, but I'm not 100% sure so it probably needs a comment.

Reply:
Deleted. Will add comments when it's back.